roofit/batchcompute/README.md

```diff
@@ -8,23 +8,23 @@ While fitting, a significant amount of time and processing power is spent on com
+This library is an internal component of RooFit, as a results users are not supposed to actively interact with it. Instead, they can benefit from significantly faster times for fitting by calling `fitTo()` and providing a `BatchMode("cpu")` or a `BatchMode("cuda")` option. 
 ```c++
-  // fit using the most efficient library that the computer's cpu can support
+  // fit using the most efficient library that the computer's CPU can support
   RooMyPDF.fitTo(data, BatchMode("cpu")); 
   
-  // fit using the cuda library along with the most efficient library that the computer's cpu can support
+  // fit using the CUDA library along with the most efficient library that the computer's CPU can support
   RooMyPDF.fitTo(data, BatchMode("cuda")); 
 ```
-**Note: In case the system does not support vector instructions, the `RooBatchCompute::Cpu` option is guaranteed to work properly by using a generic cpu library. In contrast, users must first make sure that their system supports cuda in order to use the `RooBatchCompute::Cuda` option. If this is not the case, an exemption will be thrown.**
+**Note: In case the system does not support vector instructions, the `RooBatchCompute::Cpu` option is guaranteed to work properly by using a generic CPU library. In contrast, users must first make sure that their system supports CUDA in order to use the `RooBatchCompute::Cuda` option. If this is not the case, an exemption will be thrown.**
 
-If `RooBatchCompute::Cuda` is selected, RooFit will launch CUDA kernels for computing possibilities and potentially other intense computations. At the same time, the most efficent cpu library loaded will also handle parts of the computations in parallel with the GPU (or potentially, if it's faster all of them), thus gaining full advantage of the available hardware. For this purpose `RooFitDriver`, a newly created RooFit class (in roofitcore) takes over the task of analyzing the computations and assigning each to the correct piece of hardware, taking into consideration the performance boost or penalty that may arise with every method of computing.
+If `"cuda"` is selected, RooFit will launch CUDA kernels for computing possibilities and potentially other intense computations. At the same time, the most efficent CPU library loaded will also handle parts of the computations in parallel with the GPU (or potentially, if it's faster all of them), thus gaining full advantage of the available hardware. For this purpose `RooFitDriver`, a newly created RooFit class (in roofitcore) takes over the task of analyzing the computations and assigning each to the correct piece of hardware, taking into consideration the performance boost or penalty that may arise with every method of computing.
```

Comment:
Plus: Mention also that it supports IMT.

Reply:
Ah yes, definitely that should be mentioned in the release notes :+1: I updated the PR.